<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TracertClass</class>
 <widget class="QMainWindow" name="TracertClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracert</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1001</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>851</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1,0,5,2,2">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>IP/域名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="RIP_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>真实IP：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RIP">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(215, 215, 215);</string>
        </property>
        <property name="text">
         <string>真实IP</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tracert Now</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>621</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 28pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>可视化Tracert追踪工具</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>921</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="content" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>149</y>
       <width>921</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,2,1,1,1,1,1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="A">
        <property name="text">
         <string>Router</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="B">
        <property name="text">
         <string>Source IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="C">
        <property name="text">
         <string>Destination IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="D">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>One</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="E">
        <property name="text">
         <string>Two</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="F">
        <property name="text">
         <string>Tree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="G">
        <property name="text">
         <string>Four</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="H">
        <property name="text">
         <string>Average</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="I">
        <property name="text">
         <string>Variance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>600</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Power by：MARXCBR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Tracert.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TracertClass</receiver>
   <slot>tracerting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>646</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tracerting()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
